tidy3d/components/geometry/base.py

```diff
@@ -295,8 +296,8 @@ def intersects(
             Whether the rectangular bounding boxes of the two geometries intersect.
         """
 
-        self_bmin, self_bmax = self.bounds
-        other_bmin, other_bmax = other.bounds
+        self_bmin, self_bmax = getval(self.bounds)
+        other_bmin, other_bmax = getval(other.bounds)
```

Comment:
Why is this necessary? Shouldn't the code below still work either way?

Reply:
this might have been a relic from when I was testing some validation errors when slab_bounds were traced. maybe we won't need them now.